### DISCUSSION TITLE
Option to rotate recipe picture

### DISCUSSION THREAD

**Discussion by Das-Suppenbrot** [mealie-recipes/mealie#4780] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When I upload pictures to me recipes, they are sometimes in the wrong orientation.

### Please provide a concise description of the feature that would resolve your issue.

Therefore an option to rotate it 90 degrees every button press would be very helpfull. Something like a button should do the trick.

### Please consider and list out some caveats or tradeoffs made in your design decision

One more button, but I don't think that this will be a tradeoff.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by ChipSkylark37** [comment]
I've opened an issue for this, as this appears to be a bug:

https://github.com/mealie-recipes/mealie/issues/4681

In addition to the 'rotate' issue, I would really like the ability to 'edit' a picture, similar to when adding it to the timeline, because images often don't appear how you'd expect after they get uploaded, whether it crops the image or just doesn't display the 'nicest' portion of it.

For example, I have a full image of a quiche, I upload it, and it then results in something like this:

https://snipboard.io/pGsDt2.jpg

I have to manually edit the image BEFORE I upload it and anticipate what mealie will do with it and then based on the result, may need to edit / crop manually before uploading again.
